tasks/update_record.json

```diff
@@ -16,6 +16,18 @@
         "fields": {
             "description": "ServiceNow fields to be updated. This is a hash of <field_name> => <value>",
             "type": "Hash[Any, Any, 1]"
+        },
+        "user": {
```

Comment:
Guessing Bolt has no way of specifying a `common.json` metadata file?

Reply:
I don't think so. I couldn't find any documentation/examples/evidence of shared metadata components.